extensions/project-descriptor.md

```diff
@@ -128,33 +169,30 @@ This defines the buildpacks that a platform should use on the repo.
 
 Either an `id` or a `uri` MUST be included, but MUST NOT include both. If `uri` is provided, `version` MUST NOT be allowed.
 
-### `[[build.env]]`
+#### `io.buildpacks.build.env` (optional)
 
-Used to set environment variables at build time, for example:
+This table MAY be used to set environment variables at build time, for example:
 
 ```toml
-[[build.env]]
+[[io.buildpacks.build.env]]
 name = "JAVA_OPTS"
 value = "-Xmx1g"
 ```
 
-## `[metadata]`
-
-This table includes a some defined keys, but additional keys are not validated. It can be used to add platform specific metadata. For example:
-
-```toml
-[metadata.heroku]
-pipeline = "foobar"
-```
-
 ## Example
 
 ```toml
-[project]
+[_]
 id = "io.buildpacks.my-app"
 version = "0.1"
 
-[build]
+[_.metadata]
+foo = "bar"
+
+[_.metadata.fizz]
+buzz = ["a", "b", "c"]
```

Comment:
It might be nice to have something more realistic here so folks can get a idea of why they might want to use metadata?

Reply:
yeah, I can come up with something. This is what was in the original spec.